public/script.js

```diff
@@ -59,6 +99,14 @@ canvas.addEventListener('mouseup', function (e) {
     pencilTool.mouseUp(relativeMousePosition(e));
 });
 
+canvas.addEventListener('mouseover', function (e){
+    document.body.style.cursor = "url(style/cursors/"+_lineWidth+"pix.cur), auto";
```

Comment:
We could just draw a rectangle at the mouse position on the canvas, and hide the normal cursor